docs/source/reference/respawn/movers.rst

```diff
@@ -2,18 +2,19 @@ Movers
 ======
 
 
-movers are entites that move smoothly.
+Movers are entites that move and rotate smoothly.
 ``script_mover`` allows for smooth movement and rotation contrary to ``script_mover_lightweight`` which is not able to rotate.
 
-Functions for creating movers and examples.
+Create a Mover
+-----
 
-these function are found in ``_utility.nut``
+These functions are not properties.
```

Comment:
Im not sure what this means?

Reply:
I wanted to make sure nobody thinks the functions to create a mover are a property of an entity

Reply:
Hmm, i think this is badly worded then. I think this line can be entirely removed.